integrations/ingress-nginx/documentation.yaml

```diff
@@ -0,0 +1,90 @@
+exporter_type: included
+app_name_short: Ingress NGINX
+app_name: {{app_name_short}}
```

Comment:
Should this maybe be "{app_name_short}} Controller" ?

The first line on the webpage we're linking to says "This is the documentation for the Ingress NGINX Controller."